app/views/content/train-to-be-a-teacher/what-is-a-pgce.md

```diff
@@ -8,6 +8,19 @@ backlink: "../../"
 navigation: 20.30
 navigation_title: Postgraduate certificate in education (PGCE)
 navigation_description: Having a PGCE as well as qualified teacher status (QTS) can help you develop your teaching skills and knowledge.
+youtube_video:
+  what-teaching-taught-me:
+    id: oJjoodXrvlU
+    title: A video about what a career in teaching taught maths teacher Juvayriyah
+    orientation: :portrait
+calls_to_action:
```

Comment:
@emmaachesongray I don't think these CTAs should be here - aren't these the ones you've replaced with the block promos?